content/md/connectivity/how-to-configure-catalogs-for-apps.md

```diff
@@ -0,0 +1,109 @@
+
+# What's a catalog?
+
+With catalogs, you can configure which products you want to share with an app thanks to a product selection composed of one or several criteria that allow you to filter products.
+
+When an app uses catalogs to retrieve product data, it automatically enables the Catalogs tab and creates catalogs for this app in your PIM.
+To configure your catalogs, go to the `Connect > Connected Apps`, click on `Manage app`, and then click on `Catalogs`. 
+
+Here you access the list of all the catalogs the app can manage with a label and a status. 
+
+![Catalog list](../img/app-catalog-list.png)
+
+Click on the catalog you want to set up to access the configuration interface. 
+
+When you access a catalog, you will see two tabs:
+- `Settings`: where you can enable or disable a catalog,
+- `Product selection`: where you will configure criteria to filter your product selection.
+
+::: info
+If you can't click on `Manage apps`, please read our paragraph [Who can connect apps?](#who-can-connect-apps)
+:::
+
+# Configure a product selection
+
+In the `Product selection` tab, you can define selection criteria. They are the attributes or system fields you can use to filter your products. Altogether, the criteria will create your Product selection.
+
+![Catalog - Product selection](../img/app-catalog-product-selection.png)
+
+::: warning
+When there are no criteria, your whole PIM catalog is selected. 
+:::
+
+You will find the system fields at the top of the list and below are all the attributes gathered by attribute groups. There is a search bar to help you find the attributes. 
+
+<img class="img-responsive in-article" alt="Product selection criteria" src="../img/app-catalog-criteria-selection.png" style="max-width: 700px;">
+
+Each time you add a criterion, its color turns purple to inform you that it is already used.  
+A new line appears in the product selection.
+
+Let's see how a line is organized.
+
+1. You will find the system fields or attribute you chose in purple on the left. 
+2. Then, you have to choose an operator. Depending on the filter, the operator list changes. 
+
+The following fields vary depending on your filter. There can be up to 3 types of fields.  
+- You can have the value field. For example, if your filter is a multi-select attribute type, you can choose which options you want to filter on.
+- If your filter has a value per channel and/or per locale, the channel and locale fields appear to let you choose which locale/channel you want to filter on.
+
+If you want to remove a condition, click on the cross at the end of the line.
+
+## Available conditions
+
+To select your products, you can use the following system fields and attribute types of the PIM as conditions:
+- Status
+- Category
+- Completeness
+- Family
+- Simple select attribute
+- Text attribute
+- Measurement attribute
+- Yes/No attribute
+- Multi-select attribute
+- Number attribute
+- Text area attribute
+- Date attribute
+- Identifier 
+- Measurement
```

Comment:
are you sure about documenting all available conditions ? it's easy to forget about it if we add a new one some day.

Reply:
yep! it avoids questions to the support to know why they see or don't see an attribute in the product selection filters.